aspose/src/main/java/nl/nn/adapterframework/extensions/aspose/pipe/AmountOfPagesPipe.java

```diff
@@ -0,0 +1,76 @@
+/*
+   Copyright 2019, 2020 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.extensions.aspose.pipe;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import nl.nn.adapterframework.configuration.ConfigurationException;
+import nl.nn.adapterframework.core.IPipeLineSession;
+import nl.nn.adapterframework.core.PipeRunException;
+import nl.nn.adapterframework.core.PipeRunResult;
+import nl.nn.adapterframework.doc.IbisDoc;
+import nl.nn.adapterframework.pipes.FixedForwardPipe;
+import nl.nn.adapterframework.stream.Message;
+import com.aspose.pdf.Document;
+
+/**
+ * Returns the amount of pages of a PDF file.
+ * 
+ * 
+ * @author Laurens Mäkel
+ * @since  7.6
+ *
+ */
+public class AmountOfPagesPipe extends FixedForwardPipe {
+	private String charset = "ISO-8859-1";
```

Comment:
Why isn't this StreamUtil.DEFAULT_INPUT_STREAM_ENCODING?

Reply:
I have written this pipe for a use case where we use ISO-8859-1.

Reply:
Misschien de default op utf-8 zetten, dat is namelijk overal binnen het framework zo.

Reply:
Done